src/pages/MatchingPage/index.js

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useEffect } from 'react';
 
 import Card from '../../components/Card';
-import cards from '../../data/matching.json';
+import {matching} from '../../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
have you applied prettier here?

Reply:
You are right. I will update in next commit